components/WalletHeader.vue

```diff
@@ -134,6 +137,10 @@ $breakpoint-xs: 599px;
   }
 }
 
+.non-selectable {
```

Comment:
Where is this used?

Reply:
It's totally not being used! Great catch. I was trying out moving the mouse styling around, only to find quasar already has a class called `non-selectable` and on top of that, we didn't even need it to get the effect we wanted. Removing it now, thanks @davidlehn

Reply:
bc29027